src/components/Driver/DriverLogin.js

```diff
@@ -1,37 +1,64 @@
-import React, { useState } from 'react';
-// import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import axios from 'axios';
+import {baseUrl} from '../../utils/configs'
 
 const DriverLogin = () => {
+    const history = useHistory()
+    const [loggedIn, setLoggedIn] = useState(
+      {
+          'loggedIn': false, 'userType': 'None', 'userId': 'None'
+      })
     const [fields, setFields] = useState({
-        phone: '',
+        phone_number: '',
         password: '',
+        model_name: 'Driver',
+        remember_me: false,
         alert: ''
     })
+    
+    useEffect(() => {
+        function loginStatusChanger(data) {
+            setLoggedIn({  'loggedIn': data.loggedIn,
+                'userType': data.userType,
+                'userId': data.userId})
+        }
+        async function userInChecker() {
+            await axios.get(`http://localhost:3000/api/v1/user_in`,
```

Comment:
you may consider separating axios logic into dedicated functions to make code more declarative

Reply:
OK